test/LANDRegistry.js

```diff
@@ -1098,4 +1103,168 @@ contract('LANDRegistry', accounts => {
       await assertRevert(land.assignNewParcel(0, 3, user, sentByOperator))
     })
   })
+
+  describe('LAND ping', function() {
+    describe('setGracePeriod', function() {
+      it('should set 4 weeks grace period', async function() {
+        let gracePeriod = await land.gracePeriod()
+        gracePeriod.should.be.bignumber.equal(0)
+        await land.setGracePeriod(duration.weeks(4), sentByCreator)
```

Comment:
`duration.weeks(4)` can be stored in a variable. It is being used in a lot of places.